<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>581</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="0" colspan="3">
    <widget class="QLineEdit" name="filename_lineEdit">
     <property name="text">
      <string>Filename....</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="comboBoxReadMode">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string notr="true">Full Vert Bin</string>
     </property>
     <item>
      <property name="text">
       <string>Full Vert Bin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multi-track</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Random track</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single track</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Image</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="labelReadMode">
     <property name="text">
      <string>Read Mode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButtonAbort">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(85, 0, 0)</string>
     </property>
     <property name="text">
      <string>Abort</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="comboBoxBinning">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>1x1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2x2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4x4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8x8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonCapture">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(85, 0, 0)</string>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonLive">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(85, 0, 0)</string>
     </property>
     <property name="text">
      <string>Live</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Binning</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboBoxAcqMode">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Single</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Accumulate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kinetic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fast Kinetic</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Acq. Mode</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QComboBox" name="comboBoxTrigMode">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Internal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>External</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ExternalStart</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Trig. Mode</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QCheckBox" name="checkBoxEMMode">
     <property name="text">
      <string>EM mode</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QSpinBox" name="spinBoxEMGain">
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="specialValueText">
      <string>EM Gain</string>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButtonTakeBG">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Take BG</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelExpT">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Exposure (s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditExpT">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDiv5">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>/ 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonTimes5">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>x 5</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3" colspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10000</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="3" rowspan="3">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBoxROI">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>ROI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxCrop">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Crop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxCooler">
           <property name="text">
            <string>Cooler</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keep_shutter_open_checkBox">
           <property name="text">
            <string>Keep shutter open</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="7" column="0">
          <widget class="QLabel" name="labelCenterRow">
           <property name="text">
            <string>Center</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="spinBoxCenterRow">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QSpinBox" name="spinBoxHeight">
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelNumFrames">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Frames</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QSpinBox" name="spinBoxOffset">
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="labelTracks">
           <property name="text">
            <string>Tracks</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="spinBoxNumRows">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxCycleTime"/>
         </item>
         <item row="9" column="1">
          <widget class="QSpinBox" name="spinBoxNumAccum">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="labelCycleTime">
           <property name="text">
            <string>Cycle time</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QSpinBox" name="spinBoxTracks"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="labelNumAccum">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Accum</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="labelHeight">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="labelOffset">
           <property name="text">
            <string>Offset</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="spinBoxNumFrames">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelNumRows">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Rows</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Pixels</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLineEdit" name="randomtrack_pixels_lineEdit">
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>40,60,140,160...</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>spinBoxNumFrames</zorder>
        <zorder>labelNumFrames</zorder>
        <zorder>labelNumRows</zorder>
        <zorder>spinBoxNumRows</zorder>
        <zorder>labelCenterRow</zorder>
        <zorder>spinBoxCenterRow</zorder>
        <zorder>verticalSpacer_2</zorder>
        <zorder>labelCycleTime</zorder>
        <zorder>doubleSpinBoxCycleTime</zorder>
        <zorder>spinBoxNumAccum</zorder>
        <zorder>labelNumAccum</zorder>
        <zorder>spinBoxTracks</zorder>
        <zorder>spinBoxHeight</zorder>
        <zorder>spinBoxOffset</zorder>
        <zorder>labelTracks</zorder>
        <zorder>labelHeight</zorder>
        <zorder>labelOffset</zorder>
        <zorder>label_5</zorder>
        <zorder>randomtrack_pixels_lineEdit</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QCheckBox" name="checkBoxRemoveBG">
     <property name="text">
      <string>Remove BG</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QPushButton" name="pushButton_SaveSettings">
     <property name="text">
      <string>Save settings</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="description_plainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="plainText">
      <string>Description...</string>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Group</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QPushButton" name="pushButton_LoadSettings">
     <property name="text">
      <string>Load settings</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QPushButton" name="referesh_groups_pushButton">
     <property name="text">
      <string>Refresh Groups</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QComboBox" name="group_comboBox">
     <item>
      <property name="text">
       <string>AndorData</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="save_pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLCDNumber" name="temperature_lcdNumber">
       <property name="digitCount">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="read_temperature_pushButton">
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="live_temperature_checkBox">
         <property name="text">
          <string>Live</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
